app/src/main/java/org/simple/clinic/patient/PatientSearchResult.kt

```diff
@@ -78,23 +78,25 @@ data class PatientSearchResult(
     """
     }
 
-    @Query("""$mainQuery WHERE P.uuid IN (:uuids)""")
-    fun searchByIds(uuids: List<UUID>): Single<List<PatientSearchResult>>
+    @Query("""$mainQuery WHERE P.uuid IN (:uuids) AND P.status = :status""")
+    fun searchByIds(uuids: List<UUID>, status: PatientStatus = PatientStatus.ACTIVE): Single<List<PatientSearchResult>>
```

Comment:
Nice! I like that you've used default values here because you need to force Room to serialize it. I did not think about this.

Reply:
On a second thought, this is a hidden behavior. If I'm reading the usage of `searchByIds`, I will not know that it defaults to `ACTIVE` status. Considering that it's only used in one place, do you want to make this param mandatory and let the caller specify this status?